### PR TITLE
OC 135 and OC-145

### PR DESCRIPTION
**Pull request by mehrdadgit**

### CODE REVIEW

File: src/admin/collections/insert.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      params[paramCounter++] = data[key];
+      return `$${paramCounter}`;
+    });
+    sqlFields.push('created_at', 'updated_at');
+    sqlParams.push('now()', 'now()');
+    let
+      query = `
+        INSERT INTO ${process.env.COLLECTIONS_TABLE} (${[...sqlFields]}) values (${[...sqlParams]}) returning id;`;
+
+    // console.log(query, params);
```
Comment (ZaraGlynn) [inline src/admin/collections/insert.ts]: Should probably remove the console logs if merging to dev

File: src/admin/collections/insert.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+      let insertedObject = await t.one(query, params);
+      // If we have items 
+      if (data.items && data.items.length > 0) {
+        let toBeAdded = data.items;
+        const SQL_INSERTS: string[] = toBeAdded.map((item, index) => {
+          return `($1, $${index + 2})`;
+        });
+        let addQuery = `INSERT INTO ${process.env.COLLECTIONS_ITEMS_TABLE} (collection_id, item_s3_key) VALUES ${[...SQL_INSERTS]}`;
+        let addParams = [insertedObject.id, ...toBeAdded];
```
Comment (ZaraGlynn) [inline src/admin/collections/insert.ts]: Should a lot of these lets be consts?

Comment (matthewberryman) [reply src/admin/collections/insert.ts]: Yes I agree - use const wherever possible.

File: src/admin/collections/insert.ts (hunk 1 above)
Comment (mehrdadgit) [reply src/admin/collections/insert.ts]: This one is commented out and there are a lot of console logs in the code anyway... I would not worry too much about it...  we can create a new ticket for cleaning up console logs later..

File: src/admin/collections/insert.ts (hunk 2 above)
Comment (mehrdadgit) [reply src/admin/collections/insert.ts]: I do not think we should be this picky on this; as we have more core stuff to focus on... these are block scope variables declared (and immediately used) in a small function ( they are discarded once they are out of scope)

Comment (mehrdadgit) [reply src/admin/collections/insert.ts]: Realised @Dan-Wood has fixed it below; thanks @Dan-Wood; all happy to merge this now ?

**Pull request merged**